CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,111 @@
+# Contributor Covenant Code of Conduct
+
+Please note that this project is released with a Contributor Code of Conduct. By participating in this project you agree to abide by its terms.
+
```

Comment:
Do we want a Table of Contents here? Do we want to run the remark linter/fixer on this file?